internal/bminventory/inventory_test.go

```diff
@@ -8997,9 +8977,8 @@ var _ = Describe("AMS subscriptions", func() {
 
 			newClusterName := "ams-cluster-new-name"
 			mockOperators.EXPECT().ValidateCluster(ctx, gomock.Any())
-			mockEvents.EXPECT().AddEvent(ctx, gomock.Any(), nil, models.EventSeverityInfo, eventMsgMatcher{
-				subStrings: []string{"Updated status of cluster", newClusterName},
-			}, gomock.Any())
+			mockEvents.EXPECT().SendClusterEvent(ctx, eventstest.NewEventMatcher(
+				eventstest.WithNameMatcher(eventgen.ClusterStatusUpdatedEventName)))
```

Comment:
Can we a matcher about the message as well (e.g. should contain the cluster name) and the expected cluster-id ?
same for the other expectations using eventmatcher in this file.

Reply:
Added ClusterIdMatcher